change/@uifabric-file-type-icons-2019-12-04-16-23-06-azbanduk-6.0-stream-icon.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
+  "comment": "Change files",
```

Comment:
Looks like you ended up with an extra change file somehow? Probably delete this one.